src/App.jsx

```diff
@@ -41,38 +39,42 @@ export const App = () => {
 
       <table className="table">
         <tbody>
-          {goods.map(item => (
-            <tr
-              data-cy="Good"
-              className={cn({
-                'has-background-success-light': isGoodSelected(item),
-              })}
-              key={item}
-            >
-              <td>
-                <button
-                  onClick={() => {
-                    if (good !== item) {
-                      setGood(item);
-                    } else {
-                      setGood(null);
-                    }
-                  }}
-                  data-cy={isGoodSelected(item) ? 'RemoveButton' : 'AddButton'}
-                  type="button"
-                  className={cn('button', {
-                    'button is-info': isGoodSelected(item),
-                  })}
-                >
-                  {isGoodSelected(item) ? '-' : '+'}
-                </button>
-              </td>
+          {goods.map(item => {
+            const isGoodSelected = item === good;
+
+            return (
+              <tr
+                data-cy="Good"
+                className={cn({
+                  'has-background-success-light': isGoodSelected,
+                })}
+                key={item}
+              >
+                <td>
+                  <button
+                    onClick={() => {
+                      if (good !== item) {
+                        setGood(item);
+                      } else {
+                        setGood(null);
+                      }
```

Comment:
Avoid the if else construct here, it is better to use a ternary structure. And use your 'isGoodSelected' function here.